webserver/views/user.py

```diff
@@ -98,6 +103,31 @@ def import_data():
                            lastfm_username=lastfm_username)
 
 
+@user_bp.route("/export")
+@login_required
+def export_data():
+    filename = request.args.get("filename")
```

Comment:
We can actually just export the data, but providing user an option to give file name won't be a bad idea.